src/fiskaltrust.Launcher/Extensions/LauncherConfigurationExtension.cs

```diff
@@ -0,0 +1,15 @@
+﻿using fiskaltrust.Launcher.Common.Configuration;
+
+namespace fiskaltrust.Launcher.Extensions
+{
+    static class LauncherConfigurationExtension
+    {
+        public static void LogConfigurationWarnings(this LauncherConfiguration launcherConfiguration, Serilog.ILogger logger)
+        {
+            if (launcherConfiguration.UseLegacyDataProtection.HasValue)
```

Comment:
We'll have to check that like this:
```suggestion
            if (launcherConfiguration.Raw(l => l.UseLegacyDataProtection.HasValue))
```

If not accessed through `Raw` UseLegacyDataProtection will be set to its default value so with out Raw we always log this warning even when UseLegacyDataProtection  is not set in the config.